docs/usage/rust.md

```diff
@@ -31,9 +31,23 @@ Read the [Rust environment variables docs](https://doc.rust-lang.org/cargo/refer
 
 ## Private crate registries and private Git dependencies
 
-If any dependencies are hosted in private Git repositories, [Git Authentication for cargo](https://doc.rust-lang.org/cargo/appendix/git-authentication.html) must be set up.
-
-If any dependencies are hosted on private crate registries (i.e., not on `crates.io`), then credentials should be set up in such a way that the Git command-line is able to clone the registry index.
-Third-party crate registries usually provide instructions to achieve this.
-
-Both of these are currently only possible when running Renovate self-hosted.
+This example shows how you can use a `hostRules` configuration to configure Renovate for use with a custom private Crate using Git to pull the modules when updating `Cargo.lock`.
+All token `hostRules` with a `hostType` (e.g. `github`, `gitlab`, `bitbucket`, ... ) and host rules without a `hostType` are setup for authentication.
```

Comment:
> are setup for authentication.

This is passive. I think we mean: "You as user can set authentication by doing x."